packages/compass-components/src/components/file-input.tsx

```diff
@@ -235,15 +254,20 @@ function FileInput({
           style={{ display: 'none' }}
           // Force a re-render when the values change so
           // the component is controlled by the prop.
-          key={values ? values.join(',') : 'empty'}
```

Comment:
Looks like this key was needed for making the input behave as expected when no file is chosen, or when the file is removed then re-chosen (see video in other comment).

Reply:
@Anemy Yeah, thanks for the pointer! Updated this to bring `key` back :+1: